packages/cloud-config/oem/00_rootfs.yaml

```diff
@@ -0,0 +1,15 @@
+# Rootfs cOS OEM configuration file
+#
+# This file is part of cOS and will get reset during upgrades.
+#
+# Before you change this file manually,
+# consider copying this file to /usr/local/cloud-config or
+# copy the file with a prefix starting by 90, e.g. /oem/91_custom.yaml
+name: "Rootfs Layout Settings"
+stages:
+  rootfs:
+    - environment_file: /tmp/cos-layout
```

Comment:
It's not really important at the moment, but for the long run maybe we should keep all of those under `/etc`, even under our own folder (we already have `/etc/cos-upgrade-image` which lets configure the upgrade image, but feels like we need a proper place for those)

Reply:
This is a temporary file that is only used at boot time to read `rootfs` stage contents, that's why I thought about `/tmp`. Probably even somewhere under `/run` could make sense, so after booting it could help for debugging or verifying what was actually applied. In any case note that this gets created inside in initramfs and only used in that context, so it won't ever be part of the image itself.

I agree we should think up for a better place. Probably `/run/cos/cos-layout.env`? So this way path applies for both, before and after the switch-root.

Reply:
It's indeed fine for the moment being, was just thinking out loud, because might not be the only one that needs a proper placement.  `/run/cos/*` sounds even better :+1: